#request
uint8 payload_index
---
#response
bool result